frontend/src/components/utils.spec.js

```diff
@@ -16,4 +18,9 @@ describe('utility functions', () => {
     const rubric = heatMapData[1].dataPoints;
     expect(flatData(rubric)).toEqual(false);
   });
+
+  it('removes sections with squashRubricData', () => {
+    expect(squashedHeatMapData.length).toEqual(3);
+    expect(squashedHeatMapData[0].sectionId).toEqual(undefined);
```

Comment:
This is a good start to testing `squashRubricData`. It would be good to see a few tests that confirm that the aggregation is working as expected down to the level of the data points.

Reply:
Good point. I've added more test data, and drilled in on what is happening for this test.